src/components/chip-set/chip-set.tsx

```diff
@@ -315,51 +315,62 @@ export class ChipSet {
         return (
             <div
                 class={{
-                    'mdc-text-field': true,
-                    'mdc-text-field--disabled': this.readonly || this.disabled,
+                    'mdc-text-field mdc-text-field--outlined': true,
+                    'mdc-chip-set mdc-chip-set--input': true,
                     'force-invalid': this.isInvalid(),
+                    'mdc-text-field--disabled': this.readonly || this.disabled,
                     'has-chips': this.value.length !== 0,
                     'has-leading-icon': this.leadingIcon !== null,
                 }}
                 onClick={this.handleTextFieldFocus}
             >
-                <div class="mdc-chip-set mdc-chip-set--input">
-                    {this.value.map(this.renderInputChip)}
-                    <input
-                        type="text"
-                        id="my-text-field"
-                        disabled={this.readonly || this.disabled}
-                        class={{
-                            'mdc-text-field__input': true,
-                            hidden: this.inputHidden(),
-                        }}
-                        value={this.textValue}
-                        onBlur={this.handleInputBlur}
-                        onFocus={this.handleTextFieldFocus}
-                        onKeyDown={this.handleKeyDown}
-                        onInput={this.handleTextInput}
-                        // Some browsers emit a change event on input elements, we need to stop
-                        // that event from propagating since we are emitting our own change event
-                        onChange={this.inputFieldOnChange}
-                        placeholder={this.isFull() ? '' : this.searchLabel}
-                        readonly={this.isFull()}
-                    />
-                </div>
-                {this.renderLeadingIcon()}
-                {this.renderClearAllChipsButton()}
-                <label
+                {this.value.map(this.renderInputChip)}
+                <input
+                    type="text"
+                    id="tf-outlined my-text-field"
```

Comment:
Whitespace is an illegal character in the `id` attribute. An element can only have one id.

`my-text-field` isn't referenced anywhere, so it can be removed:

```suggestion
                    id="tf-outlined"
```